lib/util/pretty-time.ts

```diff
@@ -40,3 +41,44 @@ export function toMs(time: string): number | null {
     return null;
   }
 }
+
+const rangeRegex = regEx(/^(?<operator>(>=|<=|<|>))\s*(?<age>.*)/);
```

Comment:
```suggestion
const rangeRegex = regEx(/^(?<operator>(>=|<=|<|>))\s*(?<age>.*)$/);
```

Isn't the $ needed, otherwise it's a non-greedy match and will match nothing?

Reply:
If by nothing you mean an empty string then no because an empty string doesn't start with the operators which are required at the start in the regex.

Reply:
I don't think it will capture any `age`

Reply:
Sorry you're right, I was thinking about `.*?`